docs/components/connectors/out-of-the-box-connectors/outbound/aws-dynamodb.md

```diff
@@ -11,12 +11,12 @@ The **AWS DynamoDB Connector** allows you to connect your BPMN service with Amaz
 To use the **AWS DynamoDB Connector**, you need to have an AWS account with an access key and secret key to access DynamoDB, as well as a region where your DynamoDB instance is located. You can create an account and obtain the access and secret keys from the [AWS Console](https://aws.amazon.com/console/).
 
 :::note
-It is highly recommended to use Camunda secrets to store credentials so you don't expose sensitive information directly from the process. See [managing secrets](../../../components/console/manage-clusters/manage-secrets.md) to learn more.
+It is highly recommended to use Camunda secrets to store credentials so you don't expose sensitive information directly from the process. See [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
```

Comment:
So I do notice that we refer to using secrets in a few ways, the one utilized here, and the one utilized in the document below:
"It is highly recommended to not expose your AWS IAM credentials as plain text and instead use Camunda secrets. See [manage secrets](components/console/manage-clusters/manage-secrets.md) to learn more."

Do we have a preferred way to note this? I do like that we note *why* to use secrets in the example here. Would it be better noted as a React component if duplicated elsewhere?